### ISSUE TITLE
Need notification for cancelled runs

### ISSUE THREAD

**Issue by gabiganam** [ravsamhq/notify-slack-action#41]
I noticed that if I cancel a running action, I don't get any Slack notification about it.
I'm using `if: always()` on all of my notifications.
Am I missing something? or is this not supported?

**Comment by ravgeetdhillon** [comment]
Hey @gabiganam,

Could you please share your action configuration so that I can have a better look and suggest something?

**Comment by gabiganam** [comment]
Of course @ravgeetdhillon :)

This action for example, is a simple action that deletes a k8s namespace when a branch is deleted:
```
name: Delete namespace on branch deletion
on: delete

env:
   SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}

jobs:
  delete:
    if: github.event.ref_type == 'branch'
    env:
      REF: refs/heads/${{ github.event.ref }}

    runs-on: ubuntu-latest
    steps:
      - name: Slack notification
        if: always()
        uses: ravsamhq/notify-slack-action@v1
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          status: ${{ job.status }}
          notification_title: '1/2 - Deleting namespace...'
          message_format: ''
          footer: '<{run_url}|See Action Run> | <{repo_url}|Repo {repo}> | <{workflow_url}|{workflow} Workflow>'

     <several steps redacted here>

      - name: Slack notification
        if: always()
        uses: ravsamhq/notify-slack-action@v1
        with:
          token: ${{ secrets.GITHUB_TOKEN }}
          status: ${{ job.status }}
          notification_title: '2/2 - Namespace deleted!'
          message_format: ''
          footer: '<{run_url}|See Action Run> | <{repo_url}|Repo {repo}> | <{workflow_url}|{workflow} Workflow>'
```

BTW can you please take a look at this too? https://github.com/ravsamhq/notify-slack-action/issues/39

**Comment by ravgeetdhillon** [comment]
@gabiganam Notifications for cancelled runs are now available in [v2.1.0](https://github.com/ravsamhq/notify-slack-action/releases/tag/2.1.0). For more details, checkout the [README](https://github.com/ravsamhq/notify-slack-action#readme).

**Comment by gabiganam** [comment]
Thank you for both fixes!
